lib/container.ts

```diff
@@ -377,13 +377,23 @@ export class Container extends (EventEmitter as {
 		});
 	}
 
-	public async addFilesToContainer(
+	public addFilesToContainer(
 		tarStream: Stream.Readable,
 		destination: string,
 	): Promise<void> {
-		await this.docker
-			.getContainer(this.containerId)
-			.putArchive(tarStream, { path: destination });
+		return new Promise((resolve, reject) => {
```

Comment:
NIT: this could be an `await` instead of a `return` if you keep this function async.